src/plugins/filters/components/FiltersView.vue

```diff
@@ -90,14 +90,20 @@ export default {
         this.composition.load();
```

Comment:
if we add following in mounted, its working as expected but it needs to work with filterview, so somewhere in this file stale mutation happens :p
```
if (!this.providedObject.configuration.filters) {
       return;
}
```